daphne/src/messages/taskprov.rs

```diff
@@ -54,7 +54,7 @@ impl KeyType for VdafType {
 pub enum VdafTypeVar {
     Prio3Aes128Count,
     Prio3Aes128Sum { bit_length: u8 },
-    Prio3Aes128Histogram { buckets: Vec<u64> },
+    Prio3Aes128Histogram { len_length: u8 },
```

Comment:
* `len_length` is an odd name. Maybe just `len`?
* `u8` is probably too small. I'd go with `u32`.
* We need to add a note here that this doesn't comply with the spec because the spec is out-of-date. Also, let's link to the tracking issue here.

Reply:
For some reason I thought this was the number of bytes required to store the `len`, but indeed it's just the length itself (https://www.ietf.org/id/draft-wang-ppm-dap-taskprov-04.html#section-3-9). I moved to `u32` assuming that no one will need histograms with more than 4.3 billion buckets.